src/web/components/Core/Loading.tsx

```diff
@@ -1,3 +1,12 @@
-export function Loading() {
-  return <div>Loading...</div>;
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Perhaps this warrants it's own storybook file?

Reply:
Done.